app/boards/shields/leeloo_micro/leeloo_micro.keymap

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/outputs.h>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/rgb.h>
+
+// Layers
```

Comment:
Do you need to define these if you aren't using them in the keymap?

Reply:
@caksoylar, yes, this is my understanding as well.  In Leeloo-Micro's keymap, they are used in the numpad_layer and ble_layer.  Would you recommend I pull them out by default?

Reply:
Sorry, missed that you were using them in some of the bindings (but not all, like the `&lt` thumb keys on the base layer).